src/netius/test/extra/dhcp_s.py

```
@@ -0,0 +1,212 @@
+                server.options,
+                {
+                    netius.common.SUBNET_DHCP: dict(subnet="255.255.255.0"),
+                    netius.common.LEASE_DHCP: dict(time=60),
+                },
+            )
+            self.assertEqual(server.lease, 60)
+        finally:
+            server.cleanup()
+
+    def test_get_type(self):
+        if mock == None:
```

Ruff (0.16.2)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)